src/subdomains/supporting/support-issue/services/limit-request.service.ts

```diff
@@ -69,6 +72,12 @@ export class LimitRequestService {
 
     Util.removeNullFields(entity);
 
+    await this.supportLogService.createSupportLog(entity.supportIssue.userData, {
+      type: SupportLogType.LIMIT_REQUEST,
+      limitRequest: entity,
+      ...update,
+    });
```

Comment:
This doesn't work as far as I know. You need to use the LimitRequestLogRepo to create a limit request log.

Reply:
Hm what doesn't work in this case? In my testing it creates normal entities in support_log table 🤔

Reply:
And the type was correct?

Reply:
Yes type was LimitRequest in my case, which was right 👍